src/Reservoir.jl

```diff
@@ -32,27 +40,50 @@ function run_timestep(c::Reservoir, tt::Int)
     v = c.Variables
     p = c.Parameters
     d = c.Dimensions
-    if tt==1
-        for rr in d.reservoirs
-            v.releases[rr, tt] = p.inflows[rr,tt] - p.captures[rr, tt]
-            v.storage[rr,tt] = (1-p.evaporation[rr,tt])*p.storage0[rr] + p.captures[rr, tt]
+
+    v.inflows[:,tt] = zeros(numreservoirs); 
+    v.outflows[:,tt] = zeros(numreservoirs); 
+    rr = 1
+    for gg in 1:numgauges
+       index = vertex_index(downstreamorder[gg])
+       if isreservoir[index] > 0
+	     v.inflows[rr,tt] = p.inflowsgauges[gg, tt]*1000; 
+	     v.outflows[rr,tt] = p.outflowsgauges[gg, tt]*1000; 
+	     rr += 1
```

Comment:
This relies on reservoirs being in the same order as the downstream order of gauges, but I don't see why that would be the case.  However, `isreservoir[index]` already gives the reservoir index for a give node.  So you should just be able to say:
```
rr = isreservoir[index]
v.inflows[rr,tt] = p.inflowsgauges[gg, tt]*1000;
v.outflows[rr,tt] = p.outflowsgauges[gg, tt]*1000; 
```
or
```
v.inflows[isreservoir[index],tt] = p.inflowsgauges[gg, tt]*1000; 
v.outflows[isreservoir[index],tt] = p.outflowsgauges[gg, tt]*1000;
```

Reply:
I think if you make this change, we can merge it in.